src/msw/spinctrl.cpp

```diff
@@ -439,10 +438,16 @@ bool wxSpinCtrl::SetBase(int base)
     if ( !::SendMessage(GetHwnd(), UDM_SETBASE, base, 0) )
         return false;
 
+    // DoGetBestSize uses the base.
+    InvalidateBestSize();
+
     // Whether we need to be able enter "x" or not influences whether we should
     // use ES_NUMBER for the buddy control.
     UpdateBuddyStyle();
 
+    // Update the control text.
```

Comment:
I think this comment is useless, if it said "Update the displayed text after changing the base it uses", it would be better.

But I'm mostly interested in why do we do it in this order here, but in reverse order (`SetValue()` first, then `InvalidateBestSize()`) in wxGTK -- is there a good reason for this? If yes, could you please explain it in a comment? If not, could you please use the same order in both ports?